wiki/Twitter/fr.md

```diff
@@ -0,0 +1,10 @@
+## Nous tweetons aussi!
+
+osu! a plusieurs comptes twitter, chacun sert à quelque chose de différent. La plupart ne tweetent que peu mais vous gardent au courant de ce qui se passe dans l'écosystème d'osu!
```

Comment:
__"La plupart ne tweetent que peu"__ is (maybe?) incorrect, but it's better to say __"La plupart ne tweetent qu'un peu"__.